webpack.config.js

```diff
@@ -12,22 +12,6 @@ function createEntry(entry) {
     const commonLoadersForSassAndLess = [
         {
             loader: 'style-loader',
-            options:
-                entry === 'toolbar'
-                    ? {
-                          insert: function insertAtTop(element) {
-                              // tunnel behind the shadow root
-                              if (window.__PHGTLB_ADD_STYLES__) {
-                                  window.__PHGTLB_ADD_STYLES__(element)
-                              } else {
-                                  if (!window.__PHGTLB_STYLES__) {
-                                      window.__PHGTLB_STYLES__ = []
-                                  }
-                                  window.__PHGTLB_STYLES__.push(element)
-                              }
-                          },
-                      }
-                    : undefined,
```

Comment:
Both webpack and esbuild now load the toolbar's styles via toolbar.css. This is no longer needed.